src/index.ts

```diff
@@ -74,24 +90,19 @@ async function activateCellVim(
         await app.restored;
         await jlabCodeMirror.ensureVimKeymap();
       }
-      addedCommands = addJLabCommands(app, tracker, globalCodeMirror);
-      cellManager?.modifyCell(cellManager.lastActiveCell);
-      tracker.forEach(notebook => {
-        notebook.node.dataset.jpVimMode = 'true';
```

Comment:
The updates to jpVimMode are needed for command selectors I think

Reply:
Yep, these are moved out of the if/else, to line 101. Is that okay?

`addJLabCommands` was moved out of `updateSettings` entirely, so they only get registered once.